src/core/config/wakeup.h

```diff
@@ -63,6 +63,34 @@
 #define OPENTHREAD_CONFIG_WAKEUP_END_DEVICE_ENABLE 0
 #endif
 
+/**
+ * @def OPENTHREAD_CONFIG_WED_LISTEN_INTERVAL
+ *
+ * The default WED listen interval in microseconds.
+ */
+#ifndef OPENTHREAD_CONFIG_WED_LISTEN_INTERVAL
```

Comment:
The new ones are inconsistent with the existing one

WED vs WAKEUP_END_DEVICE

Should they all be the same?

Reply:
`WED_LISTEN_DURATION` and `WED_LISTEN_INTERVAL` are terms in the spec.

When comes to `OPENTHREAD_CONFIG_WED_RECEIVE_TIME_AFTER`, the alternative would be `OPENTHREAD_CONFIG_WAKEUP_END_DEVICE_RECEIVE_TIME_AFTER`, but I believe it doesn't really improve understandability and it's too long.